src/index.js

```diff
@@ -205,7 +193,7 @@ export default function undoable (reducer, rawConfig = {}) {
     initialState: rawConfig.initialState,
     initTypes: parseActions(rawConfig.initTypes, ['@@redux/INIT', '@@INIT']),
     limit: rawConfig.limit,
-    filter: rawConfig.filter || () => true,
+    filter: rawConfig.filter || (() => true),
```

Comment:
In my project, it will build failed for SyntaxError without the extra brackets. So I add it : )